src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="header__a">
+        <img 
+          src="images/logo.png" 
+          alt="moyo-logo" 
+          class="header__img"
+        >
+      </a>
```

Comment:
there is no block such a header, you have container
```suggestion
    <header class="container">
      <a href="#home" class="container__logo">
        <img 
          src="images/logo.png" 
          alt="moyo-logo" 
          class="header__img"
        >
      </a>
```